### PR TITLE
feat(content): MasterOfGrey Person Ship

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Person)**

## Summary
"Squirrel sounds."
-The Hai, probably.

As discussed during the review process of Hai Reveal (both before and after merge, either during stream or in discord), this PR adds @MasterOfGrey's person ship to the game after all of his year's work into the Hai, HR, and basically most content in general. And for all of his hard work adding a bajillion more lines to our code. :duh:

The pers ship is a Modified Ladybug to Grey's preference, with its defining feature being speed. The sprite uses multiple unique engine ports to gather an engine "battery" of sorts to achieve this. The thrusters have been upgraded to the top engines (apart from steering). Instead of a normal Ladybug's 3 guns and 2 turrets, Grey has opted for a 5 turret approach, with reverse thrusters replacing the guns, made up of a PDT and Warder for defensive weapons, and 3 forward offensive turrets, being 2 Repeaters and 1 Bombardment Turret. However, these 3 turrets will be replaced once better Hai Pulse Turrets are added to the game as planned. Internals have been adjusted for the build, also making use of a few Korath outfits (because everyone and their mom does that).

Due to blender issues with how the original Ladybug is set up, I could not get a good looking thumb working, but I feel it's fine since most pers ships actually do not use thumbs (nor would they ever be seen), so I've just used the normal sprite for that, which most pers ships also do. It should not be a worry for this PR. 

~~This is the part where I put the funny blame joke, right?~~

Capture2
Capture

## To-Do:
 - [ ] Have Grey approve the ship and quotes

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/endless-sky/endless-sky-assets/pull/99
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/321

### CODE REVIEW

File: data/persons.txt [hunk 1]
```diff
@@ -1698,3 +1698,108 @@ effect "fusion cannon cloud"
...
+	engine -22.5 58.5 
+	engine 29.5 59 
+	engine -29.5 59 
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells, two in the back replacing the old engine pods, and one in the front replacing the gun mounts. You'd think the person who owned the ship is a bit crazy with such dangerous modifications, but sometimes speed is king.`
```
Comment (roadrunner56) [inline data/persons.txt:1805]:
```suggestion
	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells: two in the back replacing the old engine pods, and one in the front replacing the gun mounts. You'd think the person who owned the ship is a bit crazy with such dangerous modifications, but sometimes speed is king.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/persons.txt (hunk 1 above)
Comment (Saugia) [reply data/persons.txt]: ~~Description changed but~~ Would that not be a " - "?

File: data/persons.txt [hunk 2]
```diff
@@ -1706,12 +1706,28 @@ person "MasterOfGrey"
...
+	system
+		government "Hai"
 	phrase
 		word
-			"I blame Nomadic."
 			"Why is everything Alphas?"
-			"<placeholder>"
 			"You're only a terrorist if the people experience terror."
+			"Have you finished FW story?"
+			"Things don’t have to mimic an earth creature to be not very alien."
```
Comment (roadrunner56) [inline data/persons.txt:1716]:
```suggestion
			"Things don’t have to mimic an Earth creature to be not very alien."
```

File: data/persons.txt [hunk 3]
```diff
@@ -1706,12 +1706,28 @@ person "MasterOfGrey"
...
+			"I have questions. Questions that do not yet need answers."
+			"There are no bugs in my code, only Xapleaux precursors."
+			"If I were green I would photosynthesise."
```
Comment (roadrunner56) [inline data/persons.txt:1719]:
```suggestion
			"If I were green I would photosynthesize."
```

File: data/persons.txt [hunk 4]
```diff
@@ -1706,12 +1706,28 @@ person "MasterOfGrey"
...
+			`"Trust the process" isn't a joke for no reason.`
+			"Have you reviewed Hai Reveal yet?"
+			"I gotta go get ready for work. Play nice and don’t @ me."
+			"I don’t even like yellow."
+			"I really enjoy the backstory for the Marauders."
+			"Falling into the ember waste and emerging into absolute silence for the first time in dozens of gameplay hours was profoundly moving."
```
Comment (roadrunner56) [inline data/persons.txt:1725]:
```suggestion
			"Falling into the Ember Waste and emerging into absolute silence for the first time in dozens of gameplay hours was profoundly moving."
```

File: data/persons.txt [hunk 5]
```diff
@@ -1802,4 +1818,4 @@ ship "Modified Ladybug"
 	explode "large explosion" 40
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
-	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells, two in the back replacing the old engine pods, and one in the front replacing the gun mounts. You'd think the person who owned the ship is a bit crazy with such dangerous modifications, but sometimes speed is king.`
+	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells -  two in the back replacing the old engine pods, and one in the front replacing the gun mounts. These push the Modified Ladybug to speeds only an insane captain would enjoy."
```
Comment (roadrunner56) [inline data/persons.txt]:
```suggestion
	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells - two in the back replacing the old engine pods, and one in the front replacing the gun mounts. These push the Modified Ladybug to speeds only an insane captain would enjoy."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Review by GefullteTaubenbrust2 — APPROVED** [review]
Looks good to me!

File: data/persons.txt [hunk 6]
```diff
@@ -1698,3 +1698,124 @@ effect "fusion cannon cloud"
...
+	frequency 1000
+	personality
+		heroic plunders opportunistic
+	system
+		government "Hai"
+	phrase
+		word
+			"Why is everything Alphas?"
+			"You're only a terrorist if the people experience terror."
+			"Have you finished FW story?"
```
Comment (Pointedstick) [inline data/persons.txt:1715]:
```suggestion
			"Have you finished the Free Worlds storyline yet?"
```

File: data/persons.txt [hunk 7]
```diff
@@ -1698,3 +1698,124 @@ effect "fusion cannon cloud"
...
+		"Bombardment Turret"
+		"Thermal Repeater Turret" 2
+		"Point Defense Turret"
+		"Warder Anti-Missile"
+
+		"Hai Diamond Regenerator"
+		"Hai Williwaw Cooling" 2
+		"Hai Ravine Batteries"
+		"Double Plasma Core"
+		"Large Heat Shunt"
```
Comment (MasterOfGrey) [inline data/persons.txt:1782]:
```suggestion
		"Systems Core (Medium)"
		"S-970 Regenerator"
		"Crystal Capacitor"
		"Double Plasma Core"
		"Small Heat Shunt" 2
		"Large Heat Shunt"
		"Fuel Processor"
		"Quantum Keystone"
```
+16 outfit capacity and a little more reflective of my actual outfitting approach

File: data/persons.txt [hunk 8]
```diff
@@ -1698,3 +1698,124 @@ effect "fusion cannon cloud"
...
+		"Thermal Repeater Rifle" 16
+		
+		`"Biroo" Reverse Thruster`
+		`"Bondir" Atomic Steering`
+		`"Bufaer" Atomic Thruster`
+		"Hyperdrive"
```
Comment (MasterOfGrey) [inline data/persons.txt:1788]:
```suggestion
		"Hyperdrive"
		"Jump Drive"
```
+20 capacity - matching again

File: data/persons.txt [hunk 9]
```diff
@@ -1698,3 +1698,124 @@ effect "fusion cannon cloud"
...
+		"hull" 20000
+		
+		"required crew" 16
+		"bunks" 33
+		"mass" 280
+		"drag" 5.4
+		"heat dissipation" .85
+		"fuel capacity" 900
+		"cargo space" 91
+		"outfit space" 611
```
Comment (MasterOfGrey) [inline data/persons.txt:1752]:
```suggestion
		"outfit space" 645
```
To accommodate the others.

File: data/persons.txt [hunk 10]
```diff
@@ -1698,3 +1698,124 @@ effect "fusion cannon cloud"
...
+person "MasterOfGrey"
+	government "Author"
+	frequency 1000
+	personality
+		heroic plunders opportunistic
+	system
+		government "Hai"
+	phrase
+		word
+			"Why is everything Alphas?"
```
Comment (MasterOfGrey) [inline data/persons.txt:1713]: This one is tongue-in-cheek, thought it was funny.
```suggestion
		word
			"Oh dear."
			"Ah yes."
			"Lovely."
			"Suffering."
			"Why is everything Alphas?"
```

**Review by MasterOfGrey — CHANGES_REQUESTED** [review]
Saugia been on me to review this and I figured that, within the range of things you can get with access to Hai space I should look at the outfits from the perspective of how I've actually outfitted ships for my own game. So suggested some adjustments.
Love the ship, love the design, love the phrases.

**Pull request merged**